### PR TITLE
fix some syntax and typo error in docs

### PR DESCRIPTION
**Pull request by huangyuqi**
fix some syntax and typo error in docs

### CODE REVIEW

File: docs/standalone.md [hunk 1]
```diff
@@ -3,73 +3,74 @@ Runnig Heapster standalone on a host with cAdvisor
...
 
 cAdvisor is now running (in the foreground) on http://localhost:8080/.
 
-## Starting a heapster
+## Start a heapster
 
-### Config Cadvisor source
-Heapster support two types of Cadvisor source: standalone & CoreOS; Doc of source configuration is [HERE](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md)
+### Config cAdvisor source
+Heapster supports two types of cAdvisor source: standalone & CoreOS; Doc of source configuration is [HERE](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md)
```
Comment (vishh) [inline docs/standalone.md]: nit: s/source/sources.
Highlight the options `standalone` and `coreos`.

File: docs/standalone.md [hunk 2]
```diff
@@ -86,30 +87,28 @@ The `hostsFile` parameter defines a list of hosts to poll for metrics and must b
...
 I0911 13:42:03.657739   21041 heapster.go:57] ./heapster --source=cadvisor:external?cadvisorPort=8080 --use_model=true --model_resolution=10s --cache_duration=1m --stats_resolution=1s
 I0911 13:42:03.657918   21041 heapster.go:58] Heapster version 0.17.0
 I0911 13:42:03.662594   21041 heapster.go:68] Starting heapster on port 8082
 ```
 
 ###Test it out
-If you got debug information as before, means that the heapster is working normally.You can test it by requeting its [Reatful api](https://github.com/kubernetes/heapster/blob/master/docs/model.md). You can use some restful tools to test, or just run a `curl` command, 
-
-```
+If you got debug information as before, means that the heapster is working normally.You can test it by calling its [RESTful API](https://github.com/kubernetes/heapster/blob/master/docs/model.md). You can use some restful tools to test, or just run a `curl` command (_ heapster implements gzip encoding, so we need add gunzip in command "curl"_).
```
Comment (vishh) [inline docs/standalone.md]: nit: One sentence per line. It helps with readability.

**Comment by vishh** [comment]
@huangyuqi: Just a few nits. Otherwise LGTM

**Comment by vishh** [comment]
ok to test

**Comment by huangyuqi** [comment]
@vishh , thank you so much. Would you please kindly check this again? : )

File: docs/standalone.md [hunk 3]
```diff
@@ -3,73 +3,79 @@ Runnig Heapster standalone on a host with cAdvisor
...
 
 Here is an example:
+
 ```shell
 $ ./heapster --source="cadvisor:external?cadvisorPort=4194"
 ```
 
-Because heapster needs a specified file to config external cadvisor source, we need to provide one.
-The `hostsFile` parameter defines a list of hosts to poll for metrics and must be in JSON format. See below for an example:
+Because heapster needs a specified file to config external cAdvisor source, we need to provide one.
```
Comment (vishh) [inline docs/standalone.md]: Nit: Remove trailing white space here and in other places.

Comment (huangyuqi) [inline docs/standalone.md]: Hi, @vishh , adding two white space at the end, aims to place the sentence in a new line : )

Comment (vishh) [inline docs/standalone.md]: Ah! Got it!

**Comment by vishh** [comment]
LGTM

**Pull request merged**